blog/models.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from blog_system import settings
+
+
+class User(AbstractUser):
+    pass
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="posts"
+    )
+    title = models.CharField(max_length=63)
```

Comment:
The `max_length` attribute for the `title` field in the `Post` model is currently set to 63. This might be too short for some blog post titles. Consider increasing this to a larger value, such as 255, to accommodate longer titles.